pyproject.toml

```diff
@@ -3,12 +3,90 @@
 requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2"]
 build-backend = "setuptools.build_meta"
 
+# https://peps.python.org/pep-0621/
+[project]
+name = "npe2"
+dynamic = ["version"]
+description = "napari plugin engine v2"
+readme = "README.md"
+requires-python = ">=3.8"
+license = { text = "BSD-3-Clause" }
+authors = [
+    { name = "Talley Lambert", email = "talley.lambert@gmail.com" },
+    { name = "Nathan Clack" },
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
What is blocking us from adding support of python 3.11 for npe2 (I know problems with napari itself).

Reply:
Nothing, but yes, out of scope here